core/urls.py

```diff
@@ -23,4 +23,5 @@
         view=views.ProfileUpdateView.as_view(),
         name='update_profile'),
     url(r'^jsi18n/$', javascript_catalog, name='javascript_catalog'),
+    url(r'/feedback/$', views.FeedbackView.as_view(), name='feedback'),
```

Comment:
Yep! I am not sure what I tried to say that day but I guess I tried to say the same thing the docs say here:
https://docs.djangoproject.com/pt-br/1.10/topics/http/urls/
URLs don't have the first `/`.